### ISSUE TITLE
5.3.1 crashes when EVAC included

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#782]
```
Please complete the following lines...

Application Version:5.3.1 serial
SVN Revision Number:3729
Compile Date: Wed, 08 April, 2009
Operating System:Windows XP

Describe details of the issue below:
I downloaded the update 5.3.1 to try and include the EVACUATION_MC_MODE 
option, but my evacuation models no longer work.  Any time I include any 
evacuation meshes (see line in attached file) I get the following error:

forrtl: severe (157): Program Exception - access violation
Image              PC        Routine            Line        Source
fds5.exe           007289F1  Unknown               Unknown  Unknown
fds5.exe           0071301D  Unknown               Unknown  Unknown
fds5.exe           0079F13F  Unknown               Unknown  Unknown
fds5.exe           00852A33  Unknown               Unknown  Unknown
fds5.exe           0083201E  Unknown               Unknown  Unknown
kernel32.dll       7C816FE7  Unknown               Unknown  Unknown

Fire only models are still fine though
```

Original issue reported on code.google.com by `tigana` on 2009-07-09 14:58:06

<hr>
- _Attachment: [evactestcase.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-806/comment-0/evactestcase.fds)_

**Comment by gforney** [comment]
```
Timo Korhonen will take a look at this when he returns from vacation.
```

Original issue reported on code.google.com by `mcgratta` on 2009-07-09 15:48:23

**Comment by gforney** [comment]
```
See issue 716:
http://code.google.com/p/fds-smv/issues/detail?id=716

Use FDS with SVN Revision Number: 3881

Gregor


```

Original issue reported on code.google.com by `gregor.jaeger` on 2009-07-09 16:19:40

**Comment by gforney** [comment]
```
Go to the downloads page, search for "deprecated downloads"
and try to find the svn 3881 test version (dated May 1st, 2009):

   fds_5_test_3881_win32.exe     FDS+Evac 5.3.1 executable for 32 bit Windows (SVN
r3881, see Issue 716)    Deprecated      4_Test      Executable      May 01      3.5 MB     
Windows      267

I'll hope, that this will fix the problem. I have found some more bugs,
and corrected them in the source code. If you compile by yourself, then
get the latest source code from the SVN and compile. If not, then I
should try to put a new test version on the download page, if I'll
fix the Issue 805.

Note, that the test exe 3881 is compiled with option "traceback", i.e.,
the program will tell you, where it crashes.

TimoK 
```

Original issue reported on code.google.com by `tkorhon1` on 2009-07-10 18:25:16

**Comment by gforney** [comment]
```
Well, the bugs are corrected in FDS 5.4, so this issue should be fixed.

```

Original issue reported on code.google.com by `tkorhon1` on 2009-09-24 09:39:33

**Comment by gforney** [comment]
```
Well, we are now already in FDS 5.4.2, so I'll change this to "Verified"

TimoK
```

Original issue reported on code.google.com by `tkorhon1` on 2009-11-23 07:57:50
